executor/asyncloaddata/progress.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package asyncloaddata
+
+import "encoding/json"
+
+// Progress is the progress of the LOAD DATA task.
+type Progress struct {
+	// SourceFileSize is the size of the source file in bytes. When we can't get
+	// the size of the source file, it will be set to -1.
```

Comment:
When we can't get the size of the source file? Is it when loading data from local?

Reply:
https://github.com/pingcap/tidb/pull/41871/commits/4d166e3431ec635356ec9d4ad445a725c5d4e128